mmrazor/models/losses/rkd.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from ..builder import LOSSES
+
+
+@LOSSES.register_module()
+class RelationalKD(nn.Module):
+    """PyTorch version of `Relational Knowledge Distillation.
+
+    <https://arxiv.org/abs/1904.05068>`_.
+    Args:
+        loss_weight_d (float): Weight of distance-wise distillation loss.
+            Defaults to 25.0.
+        loss_weight_a (float): Weight of angle-wise distillation loss.
+            Defaults to 50.0.
+        l2_norm (bool): Whether to normalize the model predictions before
+            calculating the loss. Defaults to True.
+    """
+
+    def __init__(self, loss_weight_d=25.0, loss_weight_a=50.0, l2_norm=True):
```

Comment:
why is `loss_weight_d` : `loss_weight_a` 25 : 50 rather than 1 : 2 ?

Reply:
The paper set ``loss_weight_d`` = 25 and ``loss_weight_a`` = 50 by default.

Reply:
OK, you are right, it seems to be 1:2 in official github